### ISSUE TITLE
https://radicle.xyz/ link to the code points to... Github?

### ISSUE THREAD

**Issue by phil-lgr** [radicle-dev/radicle-interface#314]
I'm discovering Radicle and I'm very confused on how difficult is it to find the same repos on the actual Radicle app/platform

For example, how do I find [radicle-interface](https://github.com/radicle-dev/radicle-interface) on Radicle?

**Comment by phil-lgr** [comment]
Also, how does the search function?

![image](https://user-images.githubusercontent.com/9644867/183711329-9c62a803-d0d6-45d0-a7a5-fb6e5b9b8002.png)

**Comment by phil-lgr** [comment]
Finally found it after manually clicking the suggested org (search did not help)

https://app.radicle.network/alt-clients.radicle.eth/rad:git:hnrkjajuucc6zp5eknt3s9xykqsrus44cjimy/tree/0cced197baef052aa2dae7ef5acc3bc5ac27a7d0

IMO, every Github repo should list the URL of the Radicle equivalent, isn't it ironic to only direct developers to the centralized service?

**Comment by cloudhead** [comment]
Yeah, we're in the process of re-doing the website, so that will be fixed.
